tq42/algorithm.py

```diff
@@ -38,3 +38,9 @@
 from com.terraquantum.experiment.v1.experimentrun.algorithm.ml_layers.cphn_pb2 import *
 from com.terraquantum.experiment.v1.experimentrun.algorithm.ml_layers.dhn_pb2 import *
 from com.terraquantum.experiment.v1.experimentrun.algorithm.ml_layers.quantum_layer_pb2 import *
+from com.terraquantum.experiment.v1.experimentrun.algorithm.ml_layers.custom_quantum_layer_pb2 import *
+from com.terraquantum.experiment.v1.experimentrun.algorithm.ml_layers.gate.cnot_pb2 import *
```

Comment:
this is not the correct import path is it?
isn't it `com.terraquantum.experiment.v1.experimentrun.algorithm.ml_layers.gate_cnot_pb2` after the latest changes??